includes/Classifai/Providers/Azure/Personalizer.php

```diff
@@ -97,7 +97,7 @@ public function setup_fields_sections() {
 				'description'   => sprintf(
 					wp_kses(
 						// translators: 1 - link to create a Personalizer resource.
-						__( 'Azure Cognitive Service Personalizer Endpoint, <a href="%1$s" target="_blank">create a Personalizer resource</a> in the Azure portal to get your key and endpoint.', 'classifai' ),
+						__( 'Azure AI Vision Personalizer Endpoint, <a href="%1$s" target="_blank">create a Personalizer resource</a> in the Azure portal to get your key and endpoint.', 'classifai' ),
```

Comment:
Looks like Personalizer has been renamed to [Azure AI Personalizer](https://learn.microsoft.com/en-us/azure/ai-services/personalizer/what-is-personalizer), not AI Vision.

Reply:
@dkotter I've added another commit that properly handles the updating of all the Personalizer's references to Azure AI Personalizer.

Could you please have another look?